src/components/App.css

```diff
@@ -16,6 +16,7 @@
 .editor-container {
   display: flex;
   flex: 1;
+  width: 100%;
```

Comment:
@nchevobbe  @Garbee https://github.com/devtools-html/debugger.html/pull/1976#issuecomment-278764001 In continuation to this comment, I was not sure if to remove the flex layout, so made it `width 100%` here instead.
Without `width 100%`, and the editor-container still being flex, the source-header which is flex, didn't take 100% width. Any inputs?